README.md

```diff
@@ -68,6 +68,13 @@ You should probably use this README.dist.md as a base for your project's README.
 mv README.{dist.md,md}
 ```
 
+Also, in order to improve your usage of invoke scripts, you can install console autocompletion.
+```bash
+invoke --print-completion-script=MY_SHELL > ~/.bin/invoke-completion.bash
```

Comment:
1. instead of `MY_SHELL`, you can use `$(basename $SHELL)`;
2. instead of `~/.bin/invoke-completion.bash`, you can use `/etc/bash_completion.d/invoke` (and so no need to source manually the script)